src/relay/backend/contrib/cmsisnn/scalar_to_tensor_constant.cc

```diff
@@ -117,21 +117,19 @@ class ScalarToTensorConstantMutator : public MixedModeMutator {
     for (uint32_t i = 0; i < call->args.size(); ++i) {
       Expr arg = call->args[i];
       new_args.push_back(arg);
-      if (!arg->checked_type_.defined()) {
-        continue;
-      }
-      auto* arg_type = arg->type_as<TensorTypeNode>();
-      if (arg_type->shape.size() != 0 || arg.as<ConstantNode>()) {
+      const auto* arg_var = arg.as<VarNode>();
+      const auto* arg_type = arg->checked_type_.as<TensorTypeNode>();
+      if (!arg_var || !arg_type || arg_type->shape.size() != 0) {
```

Comment:
nit : could use "->IsInstance<VarNode>()" if arg_var and arg_type is not used anymore.